test/travis_test/build_2_container.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+BUILD="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+code_dir=$(dirname $BUILD)
+code_dir=$(dirname $code_dir)
+echo "build path is $BUILD"
+echo "code path is $code_dir"
+
+# Initialize the needed mizar submodule
+# git submodule update --init --recursive
+
+# Create and Start the build contrainer
+echo "    --- images list ---"
+docker images
+echo "    --- build image ---"
+docker build -f $code_dir/build/Dockerfile -t aca_build0 .
+echo "    --- images list ---"
+docker images
+echo "    --- container list ---"
+docker ps -a
+echo "    --- create container ---"
+docker rm -f aca_PARENT || true
+docker create -v $code_dir:/mnt/host/code -it --privileged --cap-add=NET_ADMIN --cap-add=SYS_PTRACE --security-opt seccomp=unconfined --name aca_PARENT aca_build0:latest /bin/bash
+docker start aca_PARENT
+docker exec aca_PARENT bash -c "/etc/init.d/openvswitch-switch restart && \
+ovs-vswitchd --pidfile --detach"
+docker rm -f aca_CHILD || true
+docker create -v $code_dir:/mnt/host/code -it --privileged --cap-add=NET_ADMIN --cap-add=SYS_PTRACE --security-opt seccomp=unconfined --name aca_CHILD aca_build0:latest /bin/bash
+docker start aca_CHILD
+docker exec aca_CHILD bash -c "/etc/init.d/openvswitch-switch restart && \
+ovs-vswitchd --pidfile --detach"
+echo "    --- container list ---"
+docker ps -a
+exit
```

Comment:
I suspect this "exit" is causing the script to exit here without running the tests below